apps/omg_watcher/test/exit_processor/persistence_test.exs

```diff
@@ -0,0 +1,240 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.ExitProcessor.PersistenceTest do
+  @moduledoc """
+  Test focused on the persistence bits of `OMG.Watcher.ExitProcessor.Core`.
+
+  The aim of this test is to ensure, that whatever state the processor ends up being in will be revived from the DB
+  """
+
+  use ExUnitFixtures
+  use OMG.DB.Case, async: true
+
+  alias OMG.API.DevCrypto
+  alias OMG.API.State.Transaction
+  alias OMG.API.Utxo
+  alias OMG.Watcher.ExitProcessor.Core
+
+  require Utxo
+
+  @eth OMG.API.Crypto.zero_address()
```

Comment:
Maybe it is worth to define `@eth` in one place because currently in every test file or code we write 
```
@eth OMG.API.Crypto.zero_address()
```

Reply:
my proposition:
 - add `OMG.Eth.eth_pseudo_address()` as _the_ function to use to get those zero bytes, via `zero_address` of course
 - continue to use `@eth OMG.Eth.eth_pseudo_address()` for short-hand in testes

Reply:
in the end figure out that most elegantly is to have:
  - `OMG.Eth.zero_address()` (moved from `Crypto`)
  - `OMG.Eth.RootChain.eth_pseudo_address()`

Reply:
The only cavil imho I can't see a value in making zero_address somehow related to Eth (as eth_pseudo_address). Of course it will logically be Eth by convention. But from code it's just address, constant.

Reply:
> related to Eth

not sure what's meant. Do you mean `Eth` the module or ETH the currency in your comment?

Reply:
Haha, this confuses me as well. I thought it as currency ;) 
Anyway, probably `zero_address` will be less confusing...